bundles/binding/org.openhab.binding.cardio2e/src/main/java/org/openhab/binding/cardio2e/internal/code/Cardio2eRelayStates.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.binding.cardio2e.internal.code;
+
+
+/**
+ * Enumerates the various RelayStates for Cardio2e binding
+ * Also provides Cardio2e string communication protocol symbol conversion
+ * 
+ * @author Manuel Alberto Guerrero Díaz
+ * @Since 1.7.0
+ */
+
+
+public enum Cardio2eRelayStates {
+	
+	OPENED { //Opened relay is equal to appliance ON
+		{
+			symbol = 'O';
+		}
+	},
+	
+	CLOSED { //Opened is equal to appliance OFF
```

Comment:
Shouldn't this say "Closed is equal to appliance OFF" ?

Reply:
Updated changes.